pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
```

Comment:
## FOR TEMPLATE UPDATE OF RULES

Does this need to be "indented" so that it is clear that it is "under" the success criterion? 

And should it have list format to not break the table layout that things are shown in as preview?

Reply:
Discussed with Wilco. It's important to keep the indentation, even though the table in preview unfortunately breaks.